.github/workflows/_comps-workflow.yml

```diff
@@ -67,7 +67,7 @@ jobs:
 
           cd ${{ github.workspace }}
           if [[ $(grep -c "llava-tgi:" ${docker_compose_yml}) != 0 ]]; then
-              git clone https://github.com/yuanwu2017/tgi-gaudi.git && cd tgi-gaudi && git checkout v2.0.4 && cd ../
+              git clone https://github.com/huggingface/tei-gaudi && cd tgi-gaudi && git checkout v2.0.4 && cd ../
```

Comment:
It's not `tei-gaudi`, it's `tgi-gaudi`. ⁠If this official build works, we'd need to build a separate `opea/llava-tgi`, just use the official one `ghcr.io/huggingface/tgi-gaudi:2.0.4`

Reply:
Suyue is right. Just use 'ghcr.io/huggingface/tgi-gaudi:2.0.4'.